owl/SOMA-SAY.owl

```diff
@@ -209,20 +493,107 @@ Note that in a semantic sense such clauses always request information, but in a
         <rdfs:subClassOf>
             <owl:Restriction>
                 <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasParticipant"/>
-                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#LinguisticObject"/>
+                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/IOLite.owl#LinguisticObject"/>
             </owl:Restriction>
         </rdfs:subClassOf>
         <rdfs:comment>An action in which an agent uses some actuator for communication purposes.</rdfs:comment>
     </owl:Class>
     
 
 
-    <!-- http://www.ease-crc.org/ont/SOMA.owl#PhrasalObject -->
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#MixingTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#MixingTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ExecutableSchematicTheory"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Instrument"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Patient"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>A schematic theory about how an agent can mix a fluid or particulate object.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#PathRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#PathRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The path of a motion is represented by a Source-Path-Goal schema.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#PhysicalObjectRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#PhysicalObjectRole">
```

Comment:
The name of the role seems not intuitive to me.
Could we find something more obvious? like *Existent* or so?

Reply:
another thought: isn't it the case that social objects can also stop existing? i.e. when the knowledge is "forgotten"

Reply:
I think a colloquial way to understand the idea behind "object permanence" in psychology (and the corresponding ESTSchemaTheories in SOMA) is the good old proverb "reality is that which, when you stop believing in it, doesn't go away".

As you say, Social Objects are obviously very dependent on the observer (though even there it is more related to the consensus of observers -- I can stop believing in money, but the German government will still demand I pay taxes). But I think, ultimately, we don't want to go there with this role and schema pair. We really aim to capture "(physical) object permanence", which is a very basic, but apparently not innate or at least not immediately manifested, cognitive ability.